ansible/inventory.yml

```diff
@@ -35,7 +35,6 @@ hosts:
         centos7-x64-1: {ip: 138.68.12.105}
 
     - ibm:
-        aix71-ppc64_be-1: {ip: 129.33.196.199, user: b9s010a}
         aix71-ppc64_be-2: {ip: 169.48.19.173}
```

Comment:
Should this be remove if you're removing the reference below

Reply:
Although with a different IP...

Reply:
No, these are release machines and the ones below are separate test machines.

Reply:
The second one needs to stay.  Its different because it is under the `release` tree as opposed to the `test` tree which is the case for the ones lower down.